package.json

```diff
@@ -49,7 +49,7 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.4.0",
     "prettier": "^2.3.0",
-    "pump": "^1.0.2",
+    "pump": "^2.0.0",
```

Comment:
@legobeat Could we use `stream` / `readable-stream` instead?

`pump` should equal `pipeline` and `end-of-stream` should equal `finished`

https://github.com/nodejs/readable-stream/blob/main/lib/stream.js#L100-L103